src/sentry/static/sentry/app/components/events/sdkUpdates.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
+
+import {t} from 'app/locale';
+
+class EventSdkUpdateSuggestion extends React.Component {
+  static propTypes = {
+    suggestion: PropTypes.object.isRequired,
+  };
+
+  getSuggestionTitle() {
+    const {suggestion} = this.props;
+    switch (suggestion.type) {
+      case 'updateSdk':
+        return <a href={suggestion.sdkUrl}>{t('update your SDK')}</a>;
+      case 'changeSdk':
+        return (
+          <a href={suggestion.sdkUrl}>
+            {t("migrate to the '%s' SDK", suggestion.newSdkName)}
+          </a>
+        );
+      case 'enableIntegration':
+        return (
+          <a href={suggestion.integrationUrl}>
+            {t("enable the '%s' integration", suggestion.integrationName)}
+          </a>
+        );
+      default:
+        return null;
+    }
+  }
+
+  render() {
+    const {suggestion} = this.props;
+    const title = this.getSuggestionTitle();
+
+    if (suggestion.enables.length == 0) {
+      return title;
+    }
+
+    return (
+      <span>
```

Comment:
should probably be a `div`

Reply:
If I make this a div I get:

```
We recommend to
Update your SDK so you can:
   ...
```

instead of:

```
We recommend to Update your SDK so you can:
   ...
```